src/DynamoIcons/DynamoIcons.csproj

```diff
@@ -1,82 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)/Config/CS.props" />
-  </ImportGroup>
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{64E226D7-2AB6-4DC2-9648-D4C34DBE2585}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>DynamoIcons</RootNamespace>
-    <AssemblyName>DynamoIcons</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="AnalysisImages.resx" />
-    <EmbeddedResource Include="DynamoUnitsImages.resx" />
-    <EmbeddedResource Include="UnitsUIImages.resx" />
-    <EmbeddedResource Include="DynamoCoreImages.resx" />
-    <EmbeddedResource Include="DynamoWatch3DImages.resx" />
-    <EmbeddedResource Include="DSCoreNodesUIImages.resx" />
-    <EmbeddedResource Include="DSCoreNodesImages.resx" />
-    <EmbeddedResource Include="DSIronPythonNodeImages.resx" />
-    <EmbeddedResource Include="DSOfficeImages.resx" />
-    <EmbeddedResource Include="ProtoGeometryImages.resx" />
-    <EmbeddedResource Include="TessellationImages.resx" />
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="System.Drawing" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="BeforeBuild" Condition=" '$(OS)' != 'Unix' ">
-    <GetReferenceAssemblyPaths TargetFrameworkMoniker=".NETFramework, Version=v2.0">
-      <Output TaskParameter="FullFrameworkReferenceAssemblyPaths" PropertyName="FrameworkAssembliesPath" />
-    </GetReferenceAssemblyPaths>
```

Comment:
What for should I do it? It's deleted file `DynamoIcons.csproj`. The whole DynamoIcons should be deleted.